process-manager/process-manager.js

```diff
@@ -20,7 +20,7 @@ ProcessManager.prototype.start = function(cb) {
   var pathToPm = require.resolve('strong-pm/bin/sl-pm');
   var args = [pathToPm, '--listen', '0', '--no-control'];
   var envOverrides = {
-    PORT: '3001'
+    PORT: '2999'
```

Comment:
This is designed to override the defined port, since that's what `slc start` will use -- which will cause a conflict. Obviously if the assigned port is 2999, there will still be issues but that will occur for any hard-coded port (and currently we don't have support for using ephemeral ports). Originally we just 3001 to avoid the most likely conflict, but with the multi-app PM, the port used is configured port + service instance, so basically 3000-3100 is off-limits.